lib/cli.js

```diff
@@ -55,17 +55,32 @@ function compileFile(filePath) {
       const fileSource = data.toString()
       let moduleName
       let bsCode
+      let diagnosticErrors 
 
       try {
         const result = ReasonablyTyped.compile(
           fileSource,
           fileName,
-          false,
+          true,
```

Comment:
Since `cli.js` aim to write reason definition into file, I think it would be appropriate to call `compile` function along to `includeModule` mode :pray: